### ISSUE TITLE
Convert password to MD5

### ISSUE THREAD

**Issue by rowland007** [rowland007/Zoo-Infrastructure#3]
Converts the password using a _message digest five_ **MD5** hash.

It is **not** required to write the MD5 from scratch. A MD5 java file will be provided.
- [x] Convert password in memory to MD5
- [x] Assign MD5 hash back to password variable

**Comment by rowland007** [comment]
### **COMPLETE**
Code has been written but needs to be tested.  Will close out this issue and if MD5 fails testing, will open a new issue in bug tracker.  Testing should be completed before this is pushed to `master` branch.
